ydb/docs/en/core/yql/reference/syntax/alter_table/changefeed.md

```diff
@@ -25,7 +25,9 @@
 {% endif %}
 * `VIRTUAL_TIMESTAMPS`: Enabling/disabling [virtual timestamps](../../../../concepts/cdc.md#virtual-timestamps). Disabled by default.
 * `RETENTION_PERIOD`: [Record retention period](../../../../concepts/cdc.md#retention-period). The value type is `Interval` and the default value is 24 hours (`Interval('PT24H')`).
-* `TOPIC_MIN_ACTIVE_PARTITIONS`: [The number of topic partitions](../../../../concepts/cdc.md#topic-partitions). By default, the number of topic partitions is equal to the number of table partitions.
+* `TOPIC_AUTO_PARTITIONING`: [Topic autopartitioning mode](../../../../concepts/cdc.md#topic-partitions):
+    * `ENABLED` - [autopartitioned topic](../../../../concepts/topic.md#autopartitioning) will be create for this changefeed. Number of partitions in such topic grows automatically as table update rate increases. Topic autopartitioning parameters [can be configured](../alter-topic.md#alter-topic).
```

Comment:
Is there `DISABLED` too? In that case, we need to specify both and mark one as the default.